runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/FarmingTracker.java

```diff
@@ -627,7 +661,76 @@ void sendNotification(RuneScapeProfile profile, PatchPrediction prediction, Farm
 		stringBuilder.append(patch.getRegion().isDefinite() ? "the " : "")
 			.append(patch.getRegion().getName())
 			.append('.');
+	}
 
-		notifier.notify(stringBuilder.toString());
+	public static PatchState GeneratePatchState(int cropItemID, int currentStage, int maxStage, int flags)
```

Comment:
should be package private
don't use pascal case 
probably should live in GeomancyManager
probably should be called something like `convertGeomancyData`

Reply:
addressed in 8677efbefdebc23f5d16b753d3d7a8652a221f3f